src/services/ssm.service.ts

```diff
@@ -12,8 +12,12 @@ export interface Parameters {
 export type GetParameters = () => Reader<AWS.SSM, Promise<Parameters>>
 
 export const getParameters: GetParameters = () => async (ssmClient) => {
-  // TODO Change ssmParameterStoreName for your project
-  const ssmParameterStoreName = '/default/node_example_lambda'
+  // We need to pull one env var from environ. The rest should come from ssm
+  // eslint-disable-next-line no-process-env
+  const nodeEnv = process.env.NODE_ENV
+  const ssmParameterStoreName = `/${
+    nodeEnv != null ? nodeEnv : 'dev'
```

Comment:
Fallback value for `NODE_ENV` should be `development`

```suggestion
    nodeEnv != null ? nodeEnv : 'development'
```

Reply:
`dev` is the convention infra-legacy uses for all its environments.

Reply:
Are we sure that's NODE_ENV and not APP_ENV?